mailer/postmark.py

```
@@ -0,0 +1,91 @@
+            items.append(
+                {
+                    "Name": item.name,
+                    "Content": content,
+                    "ContentType": mimetypes.guess_type(item.name)[0]
+                    or "application/octet-stream",
+                }
+            )
+    return items
+
+
+def send_with_postmark(
```

Pylint (3.3.7)

[refactor] Too many arguments (12/5)

(R0913)